sdk/python/foundation-models/system/content-safety/deploy_llama_with_content_safety.ipynb

```diff
@@ -0,0 +1,503 @@
+    "\n",
+    "print(\n",
+    "    f\"Choose a new Azure AI Content Safety resource in {aacs_location} with sku {aacs_sku_name}\"\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Create Azure AI Content Safety"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.mgmt.cognitiveservices.models import Account, Sku, AccountProperties\n",
+    "\n",
+    "\n",
+    "parameters = Account(\n",
+    "    sku=Sku(name=aacs_sku_name),\n",
+    "    kind=kind,\n",
+    "    location=aacs_location,\n",
+    "    properties=AccountProperties(\n",
+    "        custom_sub_domain_name=aacs_name, public_network_access=\"Enabled\"\n",
+    "    ),\n",
+    ")\n",
+    "# How many seconds to wait between checking the status of an async operation.\n",
+    "wait_time = 10\n",
+    "\n",
+    "\n",
+    "def find_acs(accounts):\n",
+    "    return next(\n",
+    "        x\n",
+    "        for x in accounts\n",
+    "        if x.kind == \"ContentSafety\"\n",
+    "        and x.location == aacs_location\n",
+    "        and x.sku.name == aacs_sku_name\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "try:\n",
+    "    # check if AACS exists\n",
+    "    aacs = acs_client.accounts.get(resource_group, aacs_name)\n",
+    "    print(f\"Found existing Azure AI content safety Account {aacs.name}.\")\n",
+    "except:\n",
+    "    try:\n",
+    "        # check if there is an existing AACS resource within same resource group\n",
+    "        aacs = find_acs(acs_client.accounts.list_by_resource_group(resource_group))\n",
+    "        print(\n",
+    "            f\"Found existing Azure AI content safety Account {aacs.name} in resource group {resource_group}.\"\n",
+    "        )\n",
+    "    except:\n",
+    "        print(f\"Creating Azure AI content safety Account {aacs_name}.\")\n",
+    "        acs_client.accounts.begin_create(resource_group, aacs_name, parameters).wait()\n",
+    "        print(\"Resource created.\")\n",
+    "        aacs = acs_client.accounts.get(resource_group, aacs_name)\n",
+    "\n",
+    "\n",
+    "aacs_endpoint = aacs.properties.endpoint\n",
+    "aacs_resource_id = aacs.id\n",
+    "print(f\"AACS endpoint is {aacs_endpoint}\")\n",
+    "print(f\"AACS ResourceId is {aacs_resource_id}\")\n",
+    "\n",
+    "aacs_access_key = acs_client.accounts.list_keys(\n",
+    "    resource_group_name=resource_group, account_name=aacs.name\n",
+    ").key1\n",
+    "print(f\"AACS access key is {aacs_access_key}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Create Azure AI Content Safety enabled LLaMA online endpoint"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.1 Check if LLaMA model is available in the AML registry."
```

Comment:
I'm ok to leave this cell in but I don't think we need to keep it, it should be guaranted.
If we keep it, a nit comment I have is use models.get() instead of models.list() then searching on client side